test/dpul_collections_web/live/search_live_test.exs

```diff
@@ -114,6 +114,48 @@ defmodule DpulCollectionsWeb.SearchLiveTest do
            |> Enum.empty?()
   end
 
+  test "when sorting by date, a nil date always sorts last", %{conn: conn} do
+    # {"id": "nildate", "title_txtm": "Document Nil Date"},
+    # {"id": "emptydate", "title_txtm": "Document Empty Date", "years_is": []}
+    Solr.add(
+      [
+        %{
+          id: "nildate",
+          title_txtm: "Document-nildate"
+        },
+        %{
+          id: "emptydate",
+          title_txtm: "Document-emptydate",
+          years_is: []
+        }
+      ],
+      active_collection()
+    )
+
+    Solr.commit()
+
+    {:ok, view, html} = live(conn, "/search?sort_by=date_desc&page=11")
+    {:ok, document} = Floki.parse_document(html)
+
+    assert document
```

Comment:
Suggestion: Can we use `has_element?` instead of Floki parsing the HTML? https://hexdocs.pm/phoenix_live_view/Phoenix.LiveViewTest.html#has_element?/3